components/admin_console/multiselect_settings.jsx

```diff
@@ -67,15 +79,15 @@ export default class MultiSelectSetting extends React.Component {
             >
                 <ReactSelect
                     id={this.props.id}
-                    multi={true}
-                    labelKey='text'
+                    isMulti={true}
+                    getOptionLabel={({text}) => text}
```

Comment:
Maybe make this a method to reduce re-renders?

Reply:
Can do.

Reply:
Wouldn't it just be `getOptionLabel={this.getOptionLabel}`?